kernel/src/managers/task/task_core.h

```diff
@@ -50,6 +50,20 @@ static inline const taskh_t *ktaskh_to_taskh(const ktaskh_t * const kth) {
     return converter.th;
 }
 
+#if CONFIG_HAS_GPDMA
+/** @struct queue of GPDMA input events
+ *
+ * @note the event field, being an enumerate, is kept fixed u32 size to avoid
+ * any compiler-specific size variation, keeping 64b size length whatever the
+ * compiler is.
```

Comment:
> keeping 64b size length whatever the compiler is.

One could be paranoid and adds a static assert here.

Reply:
added :)